pandas/_libs/tslibs/offsets.pyx

```diff
@@ -954,6 +963,27 @@ cdef class Tick(SingleConstructorOffset):
         return True
 
     def is_anchored(self) -> bool:
+        # GH#55388
+        """
+        Return False.
+
+        .. deprecated:: 2.2.0
+            The is_anchored is deprecated and will be removed in a future version.
+            Do ``False`` instead of ``offsets.Tick().is_anchored()``.
+
+        Examples
+        --------
+        >>> pd.offsets.Tick().is_anchored()
```

Comment:
Tick should never be initialized directly; use a subclass here

Reply:
thank you, I will use Hour